change/@fluentui-react-next-2020-07-30-18-07-42-a11y-menu.json

```diff
@@ -0,0 +1,8 @@
+{
+  "type": "prerelease",
+  "comment": "update snapshots",
+  "packageName": "@fluentui/react-next",
```

Comment:
once we finalized fixes in dropdown.styles.ts , we need to make the same changes in this packge too

Reply:
got it!

Reply:
Your changes look great! pls apply the same in `react-next`